### PR TITLE
Finished PickUp subsystem

### PR DESCRIPTION
**Pull request by ChrisBeeler627**
Subsystem finished: starts and stops.

### CODE REVIEW

File: src/Subsystems/PickUp.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#ifndef PickUp_H
+#define PickUp_H
+
+#include "Commands/Subsystem.h"
+#include "WPILib.h"
+
+class PickUp: Subsystem {
+
+private:
```
Comment (catvec) [inline src/Subsystems/PickUp.h]: Remove this

File: src/Subsystems/PickUp.cpp [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#include "PickUp.h"
+#include "../RobotMap.h"
+
+PickUp::PickUp(): Subsystem("PickUp"){
+	motors = new Talon(RobotMap::INTAKE_MOTOR);
+
+
```
Comment (catvec) [inline src/Subsystems/PickUp.cpp]: Remove extra space

File: src/Subsystems/PickUp.cpp [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#include "PickUp.h"
+#include "../RobotMap.h"
+
+PickUp::PickUp(): Subsystem("PickUp"){
```
Comment (ROODAY) [inline src/Subsystems/PickUp.cpp]: Add space before `{`

File: src/Subsystems/PickUp.h [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+public:
+	Talon* motors;
+
+	PickUp();
+
+	void Start();
+	void Stop();
+
+};
```
Comment (ROODAY) [inline src/Subsystems/PickUp.h]: Remove semicolon after class declaration

File: src/Subsystems/PickUp.h [hunk 5]
```diff
@@ -0,0 +1,19 @@
+#ifndef PickUp_H
+#define PickUp_H
+
+#include "Commands/Subsystem.h"
+#include "WPILib.h"
+
+class PickUp: Subsystem {
+
+public:
+	Talon* motors;
```
Comment (catvec) [inline src/Subsystems/PickUp.h]: rename to `motor`

**Comment by ROODAY** [comment]
Looks gucci fam

File: src/Subsystems/PickUp.h [hunk 6]
```diff
@@ -0,0 +1,19 @@
+#ifndef PickUp_H
+#define PickUp_H
+
+#include "Commands/Subsystem.h"
+#include "WPILib.h"
+
+class PickUp: Subsystem {
```
Comment (catvec) [inline src/Subsystems/PickUp.h]: Maybe rename to something like `BallCollector`

File: src/Subsystems/BallCollector.cpp [hunk 7]
```diff
@@ -0,0 +1,14 @@
+#include "BallCollector.h"
+#include "../RobotMap.h"
+
+BallCollector::BallCollector(): Subsystem("BallCollector") {
+	motor = new Talon(RobotMap::INTAKE_MOTOR);
+
```
Comment (catvec) [inline src/Subsystems/BallCollector.cpp]: Remove line space after `motor = ...`

File: src/Subsystems/BallCollector.h [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+class BallCollector: Subsystem {
+
+public:
+	Talon* motor;
+
+	BallCollector();
+
+	void Start();
+	void Stop();
+
```
Comment (catvec) [inline src/Subsystems/BallCollector.h]: Remove space after `void Stop();`

**Pull request merged**